### DISCUSSION TITLE
rsbuild.config.ts not found?

### DISCUSSION THREAD

**Discussion by psylaux** [web-infra-dev/rsbuild#561] [Q&A] [answered]
Helloo
I'm trying to configure pluginAssetsRetry and in the docs it's saying to modify this file, but upon installation that file is not available. I also installed rsbuild/core and rsbuild/shared. Is there something else I need to do to find this file?

**Comment by chenjiahan** [comment, marked as answer]
You can create a `rsbuild.config.ts' file in your project root directory. 

For npm packages, only `@rsbuild/core` and `@rsbuild/plugin-assets-retry` are needed.

Document: https://rsbuild.dev/guide/basic/config.html

**Comment by psylaux** [comment]
thank you, this was helpful 👍
